test/ConfigPostProcessorTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * @see       https://github.com/laminas/laminas-zendframework-bridge for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-zendframework-bridge/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-zendframework-bridge/blob/master/LICENSE.md New BSD License
+ */
+
+namespace LaminasTest\ZendFrameworkBridge;
+
+use Laminas\ZendFrameworkBridge\ConfigPostProcessor;
+use PHPUnit\Framework\TestCase;
+
+class ConfigPostProcessorTest extends TestCase
+{
+    /**
+     * @return iterable
```

Comment:
I am not sure if it is right type, always thought it should be Generator for yield

Reply:
`iterable` works for any value that can be iterated. That includes generators, iterators, and arrays. I chose `iterable` so that we can refactor this to use an array later if we want.